### PR TITLE
Release checklist update

### PR DESCRIPTION
**Pull request by cdivitotawela**
## PR description
More details on homebrew and docs releases in release checklist issue template

### Thanks for sending a pull request! Have you done the following?

- [x] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [x] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [x] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [x] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Locally, you can run these tests to catch failures early:

- [x] unit tests: `./gradlew build`
- [x] acceptance tests: `./gradlew acceptanceTest`
- [x] integration tests: `./gradlew integrationTest`
- [x] reference tests: `./gradlew ethereum:referenceTests:referenceTests`

### CODE REVIEW

**Review by macfarla — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -29,9 +29,19 @@ assignees: ''
...
-  - If publishing the release in github doesn't automatically trigger this workflow, then manually run https://github.com/hyperledger/besu-docs/actions/workflows/update-version.yml
-- [ ] Create homebrew release - run GHA workflow directly https://github.com/hyperledger/homebrew-besu/actions/workflows/update-version.yml
+- [ ] Create besu-docs release
+  - Run the GitHub workflow [update-version.yml](https://github.com/hyperledger/besu-docs/actions/workflows/update-version.yml) to update the docs version
+    - This creates a branch with name `besu-version-<version>`. PR may also be created if action has the permission
+    - If the PR has not been automatically created, create the PR manually to merge the branch to main
+    - Verify new version has been added as the stable version and last version
+    - New folder created `versioned_docs/version-<version>` [example pull request](https://github.com/hyperledger/besu-docs/pull/1675/files)
+    - Review and merge the PR to main
+  - Create a new release for the version. Copy the release notes from [hyperledger/besu](https://github.com/hyperledger/besu) to the besu-docs release
```
Comment (siladu) [inline .github/ISSUE_TEMPLATE/release-checklist.md:39]: We discussed removing this step.

**Comment by macfarla** [comment]
@siladu any input on this one?

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 2]
```diff
@@ -29,9 +29,19 @@ assignees: ''
...
+- [ ] Create homebrew release
+  - Run GHA workflow directly [update-version.yml](https://github.com/hyperledger/homebrew-besu/actions/workflows/update-version.yml)
+  - This GHA workflow will create a branch with name `update-<version>`. Workflow may create the PR if permission allowed
+  - If the PR has not been automatically created, create the PR manually to merge the branch to main
+  - Review and merge the PR to main
```
Comment (siladu) [inline .github/ISSUE_TEMPLATE/release-checklist.md:44]:
```suggestion
- [ ] Create homebrew release using [update-version workflow](https://github.com/hyperledger/homebrew-besu/actions/workflows/update-version.yml)
  - If the PR has not been automatically created, create the PR manually using the created branch `update-<version>`
```

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 3]
```diff
@@ -29,9 +29,19 @@ assignees: ''
...
-  - Copy release notes from besu
-  - If publishing the release in github doesn't automatically trigger this workflow, then manually run https://github.com/hyperledger/besu-docs/actions/workflows/update-version.yml
-- [ ] Create homebrew release - run GHA workflow directly https://github.com/hyperledger/homebrew-besu/actions/workflows/update-version.yml
+- [ ] Create besu-docs release
+  - Run the GitHub workflow [update-version.yml](https://github.com/hyperledger/besu-docs/actions/workflows/update-version.yml) to update the docs version
+    - This creates a branch with name `besu-version-<version>`. PR may also be created if action has the permission
+    - If the PR has not been automatically created, create the PR manually to merge the branch to main
+    - Verify new version has been added as the stable version and last version
+    - New folder created `versioned_docs/version-<version>` [example pull request](https://github.com/hyperledger/besu-docs/pull/1675/files)
+    - Review and merge the PR to main
```
Comment (siladu) [inline .github/ISSUE_TEMPLATE/release-checklist.md:38]:
```suggestion
- [ ] Create besu-docs release using [update-version workflow](https://github.com/hyperledger/besu-docs/actions/workflows/update-version.yml)
  - If the PR has not been automatically created, create the PR manually using the created branch `besu-version-<version>`
```

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 4]
```diff
@@ -29,9 +29,19 @@ assignees: ''
...
+  - Create a new release for the version. Copy the release notes from [hyperledger/besu](https://github.com/hyperledger/besu) to the besu-docs release
+- [ ] Create homebrew release
+  - Run GHA workflow directly [update-version.yml](https://github.com/hyperledger/homebrew-besu/actions/workflows/update-version.yml)
+  - This GHA workflow will create a branch with name `update-<version>`. Workflow may create the PR if permission allowed
+  - If the PR has not been automatically created, create the PR manually to merge the branch to main
+  - Review and merge the PR to main
+  - Run commands `brew tap hyperledger/besu && brew install besu` on MacOSX and verify latest version has been installed
```
Comment (siladu) [inline .github/ISSUE_TEMPLATE/release-checklist.md:45]: Did we discuss automating this step as well - is that still the plan?

**Review by siladu — CHANGES_REQUESTED** [review]
Although I like easy step-by-step, for this I think we benefit from keeping this checklist as succinct as possible otherwise people stop reading! Have skipped some steps that release managers should find obvious.

File: .github/ISSUE_TEMPLATE/release-checklist.md (hunk 4 above)
Comment (cdivitotawela) [reply .github/ISSUE_TEMPLATE/release-checklist.md:42]: It is in the plan and require security review from Hyperledger as well. In order to trigger this workflow from Besu release it requires a GitHub app created in the GitHub org hyperledger and have necessary permissions. GitHub organisation app need to be created by a organisations owner. I am planning to draft the architecture and validate that in a test GitHub organisation and propose the solution. If the solution is accepted and Hyperledger GitHub admins happy with the security we can proceed to implement it.

File: .github/ISSUE_TEMPLATE/release-checklist.md (hunk 1 above)
Comment (cdivitotawela) [reply .github/ISSUE_TEMPLATE/release-checklist.md:34]: I will remove the release creation as we agreed, no need to create a GitHub release.

**Review by siladu — APPROVED** [review]
LGTM

**Pull request merged**